js/currencycom.js

```diff
@@ -269,188 +266,104 @@ module.exports = class currencycom extends Exchange {
-        //     ],
-        //     symbols: [
-        //      {
-        //          symbol: "BTC/USDT",         // for LEVERAGED pairs: "BTC/USDT_LEVERAGE"
-        //          name: "Bitcoin / Tether",
-        //          status: "TRADING",          // can be: "TRADING", "BREAK", "HALT"
-        //          baseAsset: "BTC",
-        //          baseAssetPrecision: "4",
-        //          quoteAsset: "USDT",
-        //          quoteAssetId: "USDT",      // for LEVERAGED pairs: "USDT_LEVERAGE"
-        //          quotePrecision: "4",
-        //          orderTypes: [
-        //            "LIMIT",
-        //            "MARKET",
-        //            // for LEVERAGED pairs here might be "STOP" too
-        //          ],
-        //          filters: [
-        //            {
-        //              filterType: "LOT_SIZE",
-        //              minQty: "0.0001",
-        //              maxQty: "100",
-        //              stepSize: "0.0001",
-        //            },
-        //            {
-        //              filterType: "MIN_NOTIONAL",
-        //              minNotional: "5",
-        //            },
-        //          ],
-        //          marketModes: [
-        //            "REGULAR",              // can be: CLOSE_ONLY, LONG_ONLY, REGULAR
-        //          ],
-        //          marketType: "SPOT",       // for LEVERAGED pairs: "LEVERAGE"
-        //          longRate: -0.0684932,     // present in LEVERAGE pairs
-        //          shortRate: -0.0684932,    // present in LEVERAGE pairs
-        //          swapChargeInterval: 1440, // present in LEVERAGE pairs
-        //          country: "",
-        //          sector: "",
-        //          industry: "",
-        //          tradingHours: "UTC; Mon - 22:00, 22:05 -; Tue - 22:00, 22:05 -; Wed - 22:00, 22:05 -; Thu - 22:00, 22:05 -; Fri - 22:00, 23:01 -; Sat - 22:00, 22:05 -; Sun - 21:00, 22:05 -",
-        //          tickSize: "0.01",
-        //          tickValue: "403.4405",    // not present in several 'LEVERAGE' pairs (i.e. not in BTC/USDT_LEVERAGE, but in BTC/USD_LEVERAGE)
-        //          exchangeFee: "0.2",       // not present in 'LEVERAGE' pairs
-        //          tradingFee: 0.075,        // present in 'LEVERAGE' pairs
-        //          makerFee: -0.025,         // only present in a very few & major 'LEVERAGE' pairs
-        //          takerFee: 0.06,           // only present in a very few & major 'LEVERAGE' pairs
-        //          maxSLGap: 50,             // present in 'LEVERAGE' pairs
-        //          minSLGap: 1,              // present in 'LEVERAGE' pairs
-        //          maxTPGap: 50,             // present in 'LEVERAGE' pairs
-        //          minTPGap: 0.5,            // present in 'LEVERAGE' pairs
-        //          assetType: "CRYPTOCURRENCY",
-        //        },
-        //       ...
-        //   ]
-        // }
+        const response = await this.publicGetV1ExchangeInfo (params);
```

Comment:
so this was also reverted to v1 (which returns incorrect data, based i.e. 30 seconds+ earlier) ?

Reply:
It was temporarily reverted for a merge. We will restore this in consequent PRs.